packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -302,7 +299,7 @@ class CupertinoDatePicker extends StatefulWidget {
 
   /// Background color of date picker.
   ///
-  /// Defaults to [CupertinoColors.white] when null.
+  /// Defaults to null, which disables background painting entirely.
```

Comment:
Please include this detail in the issue's description, explain that we're making the change to improve fidelity wrt native iOS, and include before/after screenshots to help developers understand what's changed.

Reply:
updated